### PR TITLE
3.05: Cmake install

### PR DESCRIPTION
**Pull request by stweil**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -307,4 +307,88 @@ install(FILES
...
+    ccutil/platform.h 
+    ccutil/serialis.h 
+    ccutil/strngs.h 
+    ccutil/tesscallback.h
+    ccutil/unichar.h 
+    ccutil/unicharcompress.h 
+    ccutil/unicharmap.h 
+    ccutil/unicharset.h
+
+    #from lstm/makefile.am
```
Comment (zdenop) [inline CMakeLists.txt]: lstm in 3.05???

**Comment by stweil** [comment]
More work needed....

File: CMakeLists.txt [hunk 2]
```diff
@@ -307,4 +307,88 @@ install(FILES
     DESTINATION include/tesseract)
 
 
+get_target_property(tesseract_NAME libtesseract NAME)
```
Comment (zdenop) [inline CMakeLists.txt]: This seem to be duplicate for lines 257..307. Can you explain it please?

**Comment by stweil** [comment]
I backported the commit from master and there was a merge conflict which I did not fix correctly. Now trying an updated version...

**Comment by stweil** [comment]
It seems to work now.

**Pull request merged**